### DISCUSSION TITLE
打包带 .NET 运行时的版本

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ClassIsland/ClassIsland#545] [常规]
1. 冰点还原
2. 强行适配 Linux （？

> Wine 没测过，但是用 Steam + 应用内AloowTransparent 大体上没问题？

**Comment by Makitoid** [comment]
没必要吧。。。
电教们又都不是傻子，不会因为没安sdk发癫
再说了，就目前🇨🇳的软件生态和老师们的电脑技术，五年之内统信啥的不会替代win的

**Reply by MKStoler4096** [reply]
以备万一吧，我们已经开始杂交出现UOS了（）

**Reply by 34LiuNian** [reply]
我们这行政方面的也换UOS了

**Reply by BenYing0721** [reply]
冰点还原每次都要再装一遍确实好难受（

**Comment by Makitoid** [comment]
>我们已经开始杂交出现UOS了
啥意思？

**Reply by MKStoler4096** [reply]
就是高一用UOS，我们用Win

**Reply by Makitoid** [reply]
能冒昧问一下大佬在哪个市上学吗，教育科技这么发达

**Reply by BenYing0721** [reply]
我们这（广州）行政也开始换了

**Comment by Makitoid** [comment]
软件开发我不大了解，你的意思是用AvaloniaUI这种跨平台方案套一下吗
只有带运行时才能套AvaloniaUI？

**Reply by MKStoler4096** [reply]
原理不清楚，但是 Linux 上的 Steam 很万能……？

**Reply by Makitoid** [reply]
？

**Reply by MKStoler4096** [reply]
比如能装一些 Wine 跑不起来的玩意，很玄学（）

**Comment by doudou0720** [comment]
顶,还好我们Shadow Defender可以提交更改，不然就用不了了:(

**Comment by yeying-xingchen** [comment]
现在好像已经有了
